Future Plan.md

```diff
@@ -0,0 +1,61 @@
+# Chakracore Project Future
+
+## Future Aim
+
+Future releases of Chakracore are planned by a community team of volunteers (please see history section at the end of this document for how this has come about). We will aim to:
+
+1. Prioritise ease of use for potential embedders of chakracore - these are intended as our primary users. We will aim to reflect this priority in any design decisions and by committing to never breaking the API and providing detailed and user friendly documentation.
+
+1. Achieve full standards compliance - we recognise that Chakracore is significantly behind in some areas due to its slowed pace of development, it may take some time to catch up but we will aim to do it. Any deviation from standards should be considered a bug and we will aim to fix it.
```

Comment:
Achieving full standards compliance is unrealistic. There is no JS engine that passes all tests from test262.

Reply:
This was intending as idealistic goals not things we'd achieve in a defined period of time - though I agree that I should tone this point down, and perhaps say:

"we will work towards standards compliance including seeking to fix existing deviations and implement new standards as they arise"